src/internal.c

```diff
@@ -13279,6 +13284,10 @@ static int DoDtlsHandShakeMsg(WOLFSSL* ssl, byte* input, word32* inOutIdx,
             }
             *inOutIdx += ssl->keys.padSz;
             ret = 0;
+            /* If we receive an out of order last flight msg then retransmit */
+            if (type == server_hello_done || ret == finished) {
```

Comment:
ret is never going to be equal to finished here. Did you mean type?

Reply:
You are correct. This should be `type` not `ret`. Thanks for catching that.